test/Writer/FeedTest.php

```diff
@@ -63,45 +63,54 @@ public function testAddsAuthorUriFromArray()
 
     public function testAddAuthorThrowsExceptionOnInvalidNameFromArray()
     {
+        $this->expectException(
+            Writer\Exception\InvalidArgumentException::class
+        );
```

Comment:
minor: should be in one line, imo

Reply:
I agree, when it fits within line limit. Up to personal preference imo.